src/Containers/DistrictContainer/index.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+import React from 'react';
+import DistrictCard from '../../Components/DistrictCard';
+import { getDistricts } from '../../utils/api';
+import type { DetailedDistrictType } from '../../types';
+
+export class DistrictContainer extends React.Component {
+  state: {
+    districts: Array<DetailedDistrictType>,
+  };
+
+  constructor() {
+    super();
+    this.state = { districts: [] };
```

Comment:
ah

I have usually been defining the state outside of a constructor. So instead of

```
constructor() {
    super();
     this.state = { districts: [] };
}
```

its  just

```
state = { districts: [] };
```

less code and stuff :)